### PR TITLE
fix #37301 animaunt.tv

### LINKED ISSUE #37301: animaunt.tv
### Issue URL (Ads)
[https://animaunt.tv/1838-istoriya-finala-prodolzhenie.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fanimaunt.tv%2F1838-istoriya-finala-prodolzhenie.html)
### Comment
Username: @spok1
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.2.1
Browser: | Chrome
Stealth mode options: | Hide your search queries, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Hide your Referrer from third-parties
Filters: | AdGuard Russian,<br>AdGuard Annoyances,<br>AdGuard Base,<br>AdGuard Tracking Protection,<br>AdGuard Social Media

### PR DESCRIPTION
**Pull request by tvinzz**
https://github.com/AdguardTeam/AdguardFilters/issues/37301

### CODE REVIEW

File: AnnoyancesFilter/sections/annoyances.txt [hunk 1]
```diff
@@ -1936,6 +1936,7 @@ warezcrack.net#$#.onp-sl-content > [data-lock-id^="onpLock"] { display: block!im
 windowstechblog.nl#%#AG_onLoad(function() { window.bizpanda = {}; });
 windowstechblog.nl#$#.onp-sl-blur-area[style="filter: blur(5px);"] { filter: none!important; }
 gigadownloads.co,techsini.com##.onp-sl-overlap-box
+animaunt.tv##.icon-bar > .fl-fl
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt]: what is it? Cannot find in dev tools

File: AnnoyancesFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -84,6 +84,7 @@ run.az##div[class$="bolmem"][-ext-has='load:contains(Sayğac)']
 !--- Widgets  ---------------------!
 ! https://github.com/AdguardTeam/AdguardFilters/issues/25952
 hentaiz.org##.side > div.side-box:has(svg.fa-discord)
+animaunt.tv#?#a:contains(Стать Спонсором)
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/css_extended.txt]: make CSS part more stecific

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/annoyances.txt (hunk 1 above)
Comment (tvinzz) [reply AnnoyancesFilter/sections/annoyances.txt]: image

File: AnnoyancesFilter/sections/css_extended.txt (hunk 2 above)
Comment (tvinzz) [reply AnnoyancesFilter/sections/css_extended.txt]: changed to `animaunt.tv#?#article > .sub-info > a.sb-submit:contains(Стать Спонсором)`

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**